CMakeLists.txt

```diff
@@ -184,10 +189,18 @@ list(APPEND f3d_link_options_public ${f3d_sanitizer_link_options})
 option(BUILD_TESTING "Build the tests" OFF)
 cmake_dependent_option(F3D_TESTING_ENABLE_RENDERING_TESTS "Enable rendering tests" ON "BUILD_TESTING" OFF)
 cmake_dependent_option(F3D_TESTING_ENABLE_LONG_TIMEOUT_TESTS "Enable long timeout tests" OFF "BUILD_TESTING" OFF)
+cmake_dependent_option(F3D_TESTING_ENABLE_ONSCREEN_TESTS "Enable test that require onscreen rendering" ON "F3D_TESTING_ENABLE_RENDERING_TESTS" OFF)
 if(BUILD_TESTING)
   enable_testing()
 endif()
 
+# Testing offscreen backend
+if(NOT F3D_TESTING_OFFSCREEN_BACKEND)
+  set(F3D_TESTING_OFFSCREEN_BACKEND "auto" CACHE STRING "Testing offscreen backend" FORCE)
```

Comment:
shouldnt that be a cmake_dependent_option ?

Reply:
cannot do a `cmake_dependent_option` for types other than bool afaik